packages/date-picker/test/basic.test.js

```diff
@@ -315,25 +310,13 @@ describe('basic features', () => {
       expect(spy.called).to.be.true;
     });
 
-    describe('focus date', () => {
-      let spy;
-
-      beforeEach(() => {
-        // Do not spy on the DOM element because it can be reused
-        // by infinite scroller. Instead, spy on the native focus.
-        spy = sinon.spy(HTMLElement.prototype, 'focus');
-      });
-
-      afterEach(() => {
-        spy.restore();
-      });
-
-      it('should focus date element when opened', async () => {
-        await open(datepicker);
-        await nextRender();
-        const cell = spy.lastCall.thisValue;
-        expect(cell.hasAttribute('today')).to.be.true;
-      });
+    it('should focus date element when opened', async () => {
+      await open(datepicker);
+      await idleCallback();
```

Comment:
Locally this test passes without this, but apparently in CI it's more flaky, so I decided to add it.